CMakeLists.txt

```diff
@@ -4,7 +4,8 @@ project(maxminddb
   VERSION 1.4.3
 )
 set(MAXMINDDB_SOVERSION 0.0.7)
-add_definitions(-DPACKAGE_VERSION="${PROJECT_VERSION}")
+
+option(MAXMINDDB_BUILD_TEST_PROGRAMS "Build test programs" OFF)
```

Comment:
I have changed the default to ON so that tests/programs will be built by default.

I don't know if there's a standard, I've seen many forms of this. I can use the BUILD_TESTING option name if you prefer.

Reply:
I changed it to BUILD_TESTING.